supplementary_style_guide/glossary_terms_conventions/product_conventions/red-hat-single-sign-on.adoc

```diff
@@ -0,0 +1,146 @@
+[[red-hat-single-sign-on-conventions]]
+
+[discrete]
+[[access-token]]
+== Access token
+A token that can be provided as part of an HTTP request that grants access to the service on which the request is  being invoked. This is part of the OpenID Connect and OAuth 2.0 specification.
```

Comment:
I think the convention is to use full sentences.

Is there an extra space between "is" and "being"?

Reply:
Looks like there is a space yes, I'll fix that. Well spotted, thanks Les!
About the full sentences, I agree. It'd need to be changed throughout though for consistency.
 @andymunro would you mind weighing in on this. Should I update all the entries for full sentences or would you like to address it down the road?